core-bundle/src/Controller/FrontendController.php

```diff
@@ -107,15 +107,15 @@ public function checkCookiesAction(): Response
     public function requestTokenScriptAction(): Response
     {
         $token = $this
-            ->get('contao.csrf.token_manager')
+            ->get(ContaoCsrfTokenManager::class)
             ->getToken($this->getParameter('contao.csrf_token_name'))
             ->getValue()
         ;
 
         $token = json_encode($token);
 
         $response = new Response();
-        $response->setContent('document.querySelectorAll("input[name=REQUEST_TOKEN]").forEach(function(i){i.value='.$token.'})');
+        $response->setContent('document.querySelectorAll(\'input[name=REQUEST_TOKEN],input[name$="[REQUEST_TOKEN]"]\').forEach(function(i){i.value='.$token.'})');
```

Comment:
Why this change?

Reply:
Symfony by default creates named forms so instead of `name="REQUEST_TOKEN"` it would create something like `name="form[REQUEST_TOKEN]"` or `name="product[REQUEST_TOKEN]"`